ForgeUnity/Assets/BeardedManStudios/Scripts/Networking/Forge/Networking/Objects/NetworkObject.cs

```diff
@@ -548,6 +559,17 @@ protected virtual void OwnershipChanged()
 				ownershipChanged(Networker);
 		}
 
+		public void AssignMaster(NetworkingPlayer targetPlayer)
+		{
+			if (!IsServer)
+				return;
```

Comment:
Raise an exception or log an error here to let the caller know they're doing something wrong?